scripts/irods/test/test_pam_password_authentication.py

```diff
@@ -275,73 +280,52 @@ def test_password_expires_appropriately_based_on_grid_configuration_value(self):
         original_max_time = self.admin.assert_icommand(
             ['iadmin', 'get_grid_configuration', self.configuration_namespace, max_time_option_name], 'STDOUT')[1].strip()
 
-        IrodsController().stop()
-
         auth_session_env_backup = copy.deepcopy(self.auth_session.environment_file_contents)
-        admin_session_env_backup = copy.deepcopy(self.admin.environment_file_contents)
         try:
-            service_account_environment_file_path = os.path.join(
-                os.path.expanduser('~'), '.irods', 'irods_environment.json')
-            with lib.file_backed_up(service_account_environment_file_path):
-                client_update = test_configurations.make_dict_for_ssl_client_environment(
-                    self.server_key_path, self.chain_pem_path, self.dhparams_pem_path)
-
-                lib.update_json_file_from_dict(service_account_environment_file_path, client_update)
-
-                self.admin.environment_file_contents.update(client_update)
-
-                client_update['irods_authentication_scheme'] = self.authentication_scheme
-                self.auth_session.environment_file_contents.update(client_update)
+            client_update = {'irods_authentication_scheme': self.authentication_scheme}
+            self.auth_session.environment_file_contents.update(client_update)
 
-                with temporary_core_file() as core:
-                    core.add_rule(test_configurations.get_pep_for_ssl(self.plugin_name))
-
-                    IrodsController().start(test_mode=True)
-
-                    # Make sure the settings applied correctly...
-                    self.admin.assert_icommand(
-                        ['iadmin', 'get_grid_configuration', self.configuration_namespace, max_time_option_name],
-                        'STDOUT', original_max_time)
-
-                    # When no TTL is specified, the default value is the minimum password lifetime as configured in
-                    # R_GRID_CONFIGURATION. This value should be higher than 3 seconds to ensure steps in the test
-                    # have enough time to complete.
-                    ttl = 4
-                    self.assertGreater(ttl, 3)
-                    option_value = str(ttl)
-                    self.admin.assert_icommand(
-                        ['iadmin', 'set_grid_configuration', self.configuration_namespace, min_time_option_name, option_value])
+            # Make sure the settings applied correctly...
+            self.admin.assert_icommand(
+                ['iadmin', 'get_grid_configuration', self.configuration_namespace, max_time_option_name],
+                'STDOUT', original_max_time)
```

Comment:
This assertion seems a bit strange. I don't see where the value is being changed? I haven't spotted how this can fail.

Reply:
Removed